crypto/src/lokinet_identity.cpp

```diff
@@ -56,10 +54,15 @@ std::vector<uint8_t> parseLokinetIdentityPublic(const std::string& path) {
             "Lokinet identity.private file could not be found");
     }
     std::ifstream input(p.c_str(), std::ios::binary);
-    const std::vector<uint8_t> bytes(std::istreambuf_iterator<char>(input), {});
-    const std::vector<uint8_t> publicKey(bytes.begin() + publicKeyOffset,
-                                         bytes.begin() + publicKeyOffset +
-                                             keyLength);
+    std::array<uint8_t, keyLength> privateKey;
+    int i = 0;
+    while(i < keyLength){
+        input >> privateKey[i++];
```

Comment:
Need to check errors here (file too sort etc). Could we use `istream::read` instead?